src/onResize/addEventListeners/functions/clearHovered.js

```diff
@@ -0,0 +1,16 @@
+export default function clearHighlighted() {
+    this.lines
+        .filter(function() {
+            return !d3.select(this).classed('selected');
+        })
+        .select('path')
+        .attr(this.config.line_attributes);
+    this.points
+        .filter(function() {
+            return !d3.select(this).classed('selected');
+        })
+        .select('circle')
+        .attr(this.config.point_attributes)
+        .attr('r', this.config.marks.find(mark => mark.type === 'circle').radius);
```

Comment:
ugh - hate that this is needed to update the radius. we should clean this up in v2.0